mock/src/app.ts

```diff
@@ -0,0 +1,239 @@
+import express, { Router } from 'express';
+import cors from 'cors';
+import { MBIResourceType } from '../../src/@types/centreonAPI';
+
+function getRandomArbitrary(min: number, max: number) {
+  return Math.round(Math.random() * (max - min) + min);
+}
+
+const app = express();
+const port = 3001;
+
+app.use(cors({ origin: true }));
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+
+const router = Router();
+
+app.use('/centreon/api/latest', router);
+
+router.use((req, res, next) => {
+  try {
+    console.log(
+      `${req.method} ${req.originalUrl} \n  headers : \n\t${Object.entries(req.headers)
+        .map(([k, v]) => `${k}: ${v}`)
+        .join('\n\t')} \n  body : \n${JSON.stringify(req.body, null, 2).split('`\n').join('\n\t')}\n\n`
+    );
+  } finally {
+    next();
+  }
+});
+
+function ensureAuthenticated(req: express.Request, res: express.Response, next: Function) {
+  const token = req.header('X-AUTH-TOKEN');
+  if (token && Number(token) > Date.now()) {
+    return next();
+  } else {
+    res.status(401).send('UNAUTHORIZED');
+  }
+}
+
+router.post('/login', (req, res) => {
+  const { body } = req;
+
+  if (body?.security?.credentials?.password === 'centreon') {
+    // token valid 30 minutes to test
```

Comment:
Remove the comment and extract Date.now() + 30 * 60 * 1000 into a dedicated variable